mizar/obj/tests/test_vpc_with_dup_vni.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: mizar.com/v1
```

Comment:
We need to add tests that can leverage these yamls and verify it as part of CI.

Reply:
I have added tests to the e2e.